services/client/src/components/LogsTable.js

```diff
@@ -0,0 +1,94 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { Row, Col } from 'antd';
+
+import TableList from 'components/TableList';
+import formatDistanceToNow from '../utils/formatDistanceToNow';
+
+const LogsTable = ({ logs, pagination, loading, onClickRow, onPageChange }) => {
+  const columns = [
+    {
+      title: 'Datasource',
+      dataIndex: ['datasource', 'name'],
+      key: 'datasource_name',
+    },
+    {
+      title: 'Path',
+      dataIndex: 'path',
+      key: 'path',
+      render: (_, record) => record.request_event_logs?.find(e => e.path)?.path,
```

Comment:
is it possible to put `path` in the `request_logs` table? If you'll have 1000 request event logs then all table load will take a lot of time